packages/core/src/__tests__/error.test.ts

```diff
@@ -0,0 +1,56 @@
+import {ComponentError, logIfComponentError} from "../utils/ComponentError";
+
+const mockLogHandledError = jest.fn();
+
+jest.mock("react-native", () => ({
+  NativeModules: {
+    EmbraceManager: {
+      logHandledError: (
+        message: string,
+        componentStack: string,
+        params: object,
+      ) => {
+        mockLogHandledError(message, componentStack, params);
```

Comment:
the `mockLogHandledError` itself could return the `true`. I believe something like

```
const mockLogHandledError = jest.fn().mockReturnValue(true);
```

and after something like

```
...
logHandledError: (
        message: string,
        componentStack: string,
        params: object,
      ) => mockLogHandledError(message, componentStack, params),
...

```

and similar in other places